src/constants.ts

```diff
@@ -11,6 +11,8 @@ export namespace DialogBoxResponses {
     export const Yes: MessageItem = { title: "Yes" };
     export const OK: MessageItem = { title: "OK" };
     export const DontShowAgain: MessageItem = { title: "Don't Show Again" };
+    export const upload: MessageItem = { title: "Upload" };
+    export const uploadDontWarn: MessageItem = { title: "Upload, don't warn again" };
```

Comment:
How about "Always upload"?

Reply:
We discussed that and I think Matt preferred this message for some reason. In any case, this is consistent with our other extensions I'm not going to change the other extensions before the release